rjaltman/dogu
# Search

Implement (very simple) searching functionality. 

TO TEST:
Look at the tags that I manually made in the migrations. Make sure you can search by those. 
Make sure that you can search two non-consecutive words in a project description and still have the project show up. 

Close #10
